make.sh

```diff
@@ -110,5 +111,11 @@ rsync -av \
     --exclude '.git' \
     --exclude '.gitmodules' \
     --exclude 'make.sh' \
-    --exclude 'sample' \
+    --exclude './sample' \
     ./ sample/
+
+# Correct path in Dangerfile in sample
+#sed -i '' "s/'.\//'.\/sample\/ios\//g" ./sample/ios/dangerfile
+#sed -i '' 's/".\//".\/sample\/ios\//g' ./sample/ios/dangerfile
+#sed -i '' "s/'\.swiftlint\.yml'/'\.\/sample\/ios\/\.swiftlint\.yml'/g" ./sample/ios/dangerfile
+#sed -i '' "s/'Pods\/\*\*'/'sample\/ios\/Pods\/\*\*'/g" ./sample/ios/dangerfile
```

Comment:
Time to remove? 
https://github.com/nimblehq/kmm-templates/actions/runs/5971627252/job/16200951105?pr=23 ✅